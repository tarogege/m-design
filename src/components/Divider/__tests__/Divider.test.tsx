import { render } from "@testing-library/react";
import { Divider } from "maodesign";

describe("test divider component", () => {
  it("render a default divider", () => {
    const c = render(<Divider />);
    const tree = c.asFragment;
    expect(tree).toMatchSnapshot();
  });
});
