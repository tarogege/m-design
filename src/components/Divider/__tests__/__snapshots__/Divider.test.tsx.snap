// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test divider component > render a default divider 1`] = `[Function]`;
